atmos/main.variant

```diff
@@ -58,5 +58,5 @@ imports = [
   "./modules/helm",
   "./modules/workflow",
   "./modules/istio",
-  "./modules/vendor"
+  "./modules/vendor-command"
```

Comment:
why did we move `vendor` to `vendor-command` (I don't recollect that) ?

Reply:
@Nuru told me it's because:

> The build copies the entire source folder into the binary, so the source folder should be as small as possible.

Reply:
oh, this is different though. I don't know why we're doing that here. Then we should have `helm-command`, `terraform-command`, etc. I'd rather not change this. Or if we must, call it `vendir`

Reply:
@aknysh @osterman The folder name `vendor` is [reserved by Go](https://golang.org/cmd/go/#hdr-Vendor_Directories) for local copies of external dependencies. It causes a world of hurt to have a folder named `vendor` in any project that includes Go code. (It has even caused problems in `cloudposse/packages`) 

I thought it better to rename it `vendor-command` but I have no problem with calling it `vendir` if you prefer. LMK.

Reply:
Ok, let's call it vendir. That is more accurate anyways